dist/scripts/controllers/LandingCtrl.js

```diff
@@ -17,75 +35,42 @@
              }
              else {
                 this.sessionDisplay = this.START_SESSION;
-                this.timerManager.startSession();
-                 
+                this.timerManager.startSession();   
              }
-             
-         
          };
          
-         
+         /**
+         * @function startBreak
+         * @desc Starts or resets a break. Ensures that the user has the correct instructions.
+         */
          this.startBreak = function () {
              if (!TimerManager.running) {
                 this.breakDisplay = this.RESET;
                 this.sessionDisplay = this.START_SESSION;
-                this.timerManager.startBreak();
-                 
                 
+                 this.timerManager.startBreak();
              }
              else {
                 this.breakDisplay = this.START_BREAK;
                 
-                
                 this.timerManager.startBreak();
-                 
-                
-                 
              }
-             
-         
          };
-         
-
-          
+                 
          var currentBuzzObject = new buzz.sound('/assets/sounds/DING2', {
                 formats: ['mp3'],
                 preload: true
          });
          
-         $scope.$watch('timeMan.currentTime', function() {
+         $scope.$watch('$scope.timeMan.currentTime', function() {
              if($scope.timeMan.currentTime == 0) {
                  currentBuzzObject.play();
              }   
```

Comment:
When you're in a $watch, you don't need to pass in the $scope - so it'd just be `$scope.$watch('timeMan.currentTime'...` instead.